reference/conanfile/attributes.rst

```diff
@@ -1026,8 +1026,12 @@ to :ref:`repackage artifacts <repackage>`
 
 .. _scm_attribute:
 
-scm
----
+scm [EXPERIMENTAL]
```

Comment:
Do not put EXPERIMENTAL in titles, because later the URLs use it, and then, when it is removed, it will break URLs

Reply:
You are right. I was not aware of that and unfortunately we have experimental in a bunch of titles right now